manifests/istio-operator-template-light.yaml

```diff
@@ -0,0 +1,308 @@
+      k8s:
+        affinity:
+          podAntiAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                    - istiod
+                topologyKey: kubernetes.io/hostname
+              weight: 100
+        env:
+        - name: PILOT_HTTP10
+          value: "1"
+        hpaSpec:
+          maxReplicas: 1
+          minReplicas: 1
+        podAnnotations:
+          reconciler.kyma-project.io/managed-by-reconciler-disclaimer: |
+            DO NOT EDIT - This resource is managed by Kyma.
+            Any modifications are discarded and the resource is reverted to the original state.
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 1024Mi
+          requests:
+            cpu: 50m
+            memory: 128Mi
+        securityContext:
+          runAsGroup: "65534"
+          runAsNonRoot: true
+          runAsUser: "65534"
+          seccompProfile:
+            type: RuntimeDefault
+        serviceAnnotations:
+          reconciler.kyma-project.io/managed-by-reconciler-disclaimer: |
+            DO NOT EDIT - This resource is managed by Kyma.
+            Any modifications are discarded and the resource is reverted to the original state.
+  hub: eu.gcr.io/kyma-project/external/istio
+  meshConfig:
+    accessLogEncoding: JSON
+    accessLogFile: ""
+    defaultConfig:
+      holdApplicationUntilProxyStarts: true
+      proxyMetadata: {}
+    defaultProviders:
+      tracing: []
+    enablePrometheusMerge: false
+    enableTracing: true
+    extensionProviders:
+    - name: kyma-traces
+      opencensus:
+        context:
+        - W3C_TRACE_CONTEXT
+        port: 55678
+        service: telemetry-trace-collector-internal.kyma-system.svc.cluster.local
+    - envoyFileAccessLog:
+        logFormat:
+          labels: {}
+        path: /dev/stdout
+      name: envoy
+    - envoyFileAccessLog:
+        logFormat:
+          labels: {}
+        path: /dev/stdout
+      name: stdout-json
+    trustDomain: cluster.local
+  profile: default
+  tag: "{{ .IstioVersion }}-{{ .IstioImageBase }}"
+  values:
+    base:
+      enableCRDTemplates: false
+      validationURL: ""
+    cni:
+      cniBinDir: /opt/cni/bin
+      cniConfDir: /etc/cni/net.d
+    defaultRevision: ""
+    gateways:
+      istio-egressgateway:
+        autoscaleEnabled: true
+        env: {}
+        name: istio-egressgateway
+        secretVolumes:
+        - mountPath: /etc/istio/egressgateway-certs
+          name: egressgateway-certs
+          secretName: istio-egressgateway-certs
+        - mountPath: /etc/istio/egressgateway-ca-certs
+          name: egressgateway-ca-certs
+          secretName: istio-egressgateway-ca-certs
+        type: ClusterIP
+      istio-ingressgateway:
+        autoscaleEnabled: true
```

Comment:
This is set to false in the existing evaluation profile, did we change this on puropose?